BubblesKotlin/app/src/main/java/com/example/android/bubbles/data/NotificationHelper.kt

```diff
@@ -48,6 +51,9 @@ class NotificationHelper(private val context: Context) {
     private val notificationManager =
         context.getSystemService(NotificationManager::class.java) ?: throw IllegalStateException()
 
+    private val shortcutManager =
+        context.getSystemService(ShortcutManager::class.java) ?: throw IllegalStateException()
```

Comment:
Use the ContextCompat or [extfun](https://developer.android.com/reference/kotlin/androidx/core/content/package-summary#(android.content.Context).getSystemService()) method instead? (to make it work pre-Q). Same comment above.

Reply:
Done
I'll add more compat code once it supports R features of NotificationManager.